azure-functions/src/send_grid/message_builder.rs

```diff
@@ -583,17 +647,19 @@ impl MessageBuilder {
     /// # Examples
     ///
     /// ```rust
-    /// # use azure_functions::send_grid::MessageBuilder;
+    /// use azure_functions::bindings::SendGridMessage;
     /// use azure_functions::send_grid::Attachment;
-    /// let message = MessageBuilder::new()
+    ///
+    /// let message = SendGridMessage::build()
```

Comment:
Same as comment above for this example.

Reply:
I think I fixed this throughout.

Reply:
There's still one more "hello world" in the diff for some reason (line 615).